lib/spack/docs/config_yaml.rst

```diff
@@ -184,3 +184,12 @@ other tools may need custom ``LD_LIBRARY_PATH`` setings to run.  You can
 set ``dirty`` to ``true`` to skip the cleaning step and make all builds
 "dirty" by default.  Be aware that this will reduce the reproducibility
 of builds.
+
+--------------------
+``ccache``
+--------------------
+
+When set to ``true`` Spack will use ccache to cache compiles. This is
+useful specifically un two cases: (1) Use with ``spack setup``, (2)
+Build the same package with many different variants. The default is
+``false``.
```

Comment:
> How does Spack find ccache?

You tell me, how you want it (see my comment above). Currently spack just assumes `ccache` is in `PATH`. And I can certainly write that here.

>  Are you supposed to ask Spack to build it first?

Sure, if you don't have it on your system that is an option.

> How should ccache be configured?

`ccache` works nicely out of the box, but we can of course refer the user to the _Configuration settings_ of `man ccache`.